Framework/CurveFitting/src/Algorithms/SplineSmoothing.cpp

```diff
@@ -303,10 +324,10 @@ void SplineSmoothing::addSmoothingPoints(const std::set<int> &points,
  * @param row :: The row of spectra to use
  */
 void SplineSmoothing::selectSmoothingPoints(
-    MatrixWorkspace_const_sptr inputWorkspace, size_t row) {
+    MatrixWorkspace_const_sptr inputWorkspace, const size_t row) {
```

Comment:
Since you are doing other nice cleanup like adding `const`, you can also replace shared pointers by references in many cases, e.g., here `MatrixWorkspace_const_sptr` -> `const MatrixWorkspace &` (if you want to).